addons/web/static/src/views/view_components/view_scale_button.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.ViewScaleButton" owl="1">
+        <span t-if="Object.keys(props.scales).length > 1" class="o_view_scale_button">
+            <Dropdown class="'btn-group'" togglerClass="'btn btn-secondary scale_button_selection text-uppercase'" hotkey="'v'" showCaret="true">
+                <t t-set-slot="toggler">
+                    <t t-esc="props.scales[props.currentScale].description" />
+                </t>
+                <t t-foreach="Object.entries(props.scales)" t-as="scale" t-key="scale[0]">
+                    <DropdownItem
+                        class="`o_scale_button_${scale[0]} btn btn-secondary text-uppercase ${scale[0] === props.currentScale ? 'active' : ''}`"
```

Comment:
remove `btn btn-secondary`

with this a design issue occurs on touch devices, seek for rule `.o_web_client.o_touch_device` in `webclient.scss`
![image](https://user-images.githubusercontent.com/1159815/211534934-959774fe-33fb-4491-ba8d-3bf8dac2d6b1.png)

Reply:
Well, this makes the target much smaller without `btn` on it :/
<img width="162" alt="image" src="https://user-images.githubusercontent.com/35101914/211568482-cfffe1b5-69d9-47b8-b0c4-1126e8f0f87e.png">
vs
<img width="143" alt="image" src="https://user-images.githubusercontent.com/35101914/211568577-f13408b2-381c-4e70-a262-592934399dba.png">

Let me see how to improve this

Reply:
@brboi  Can you to look at the changes I propose in webclient.scss?
Should it affect negatively other menu items? This looks to serve a similar purpose, writtent differently. but I may be wrong

With this, the arrow is located at the center of the available space, in both touch and desktop display, but the selector has a better padding on touch devices

Reply:
Not convinced, the checkmark is too close to the border. I tried in enterprise and community as well.
![image](https://user-images.githubusercontent.com/1159815/211592944-0ed3580b-5ea7-4d65-a2a4-14a09aa5984a.png) ![image](https://user-images.githubusercontent.com/1159815/211592585-20688505-541d-417e-813f-8ddb0d223f23.png)

I think you should just remove the `btn` classes from the dropdown items.
I understand that you find too small the items for a small screen device, but maybe this could be tackled in a broader task for all dropdown items? The app menu on community version has the "small version" for a long time and I think no one has complained yet.

cc @odoo/rd-responsiveness for your precious insights

Reply:
Okay, let's get rid of the btn classes here.
Yes, I think that dropdown items are too compact. When I click with my finger, I sometimes just hit the wrong line